templates/tfengine/recipes/cicd.hcl

```diff
@@ -168,11 +168,6 @@ schema = {
 
             NOTE: The modules will be deployed in the given order. If a module
             depends on another module, it should show up after it in this list.
-
-            NOTE: The CICD has permission to update APIs within its own project.
```

Comment:
I missed the previous discussion, how come this paragraph is no longer needed?

Reply:
It's not a commonly used feature, and would actually cause more problems. I removed it from the example in https://github.com/GoogleCloudPlatform/healthcare-data-protection-suite/pull/819 but didn't remove it from the schema.